lab8_9/tests/unit/test_skip.py

```diff
@@ -0,0 +1,2 @@
+def test_try():
+    assert True
```

Comment:
so, I would suggest you to put your code to functions. because now you cannot test it